### ISSUE TITLE
0.2.3 stopped working

### ISSUE THREAD

**Issue by breadbakerman** [CrankshaftForSteam/crankshaft#8]
It comes up at start but fails to initialize SMM with this error:

(unknown) Uncaught (in promise) Error: Error initializing SMM after retry: TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'.
    at main (<anonymous>:7704:15)
    at <anonymous>:7729:3
    at <anonymous>:7730:3

**Comment by coolavery** [comment]
Thanks for the report! I accidentally published a broken 0.2.3 build to Flathub (I forgot they have auto-publishing) for a few hours before reverting it to 0.2.2. You should be able to "update" again and get back to 0.2.2, or you can manually update with:

```
flatpak --user update --commit=fd1135824af42a3492b7c0c0808c65b6eec4f25ca134a66409d28dbbe7097c7a space.crankshaft.Crankshaft
```

Sorry for the trouble, I'll be releasing 0.2.4 today which will fix the release issues, and I'm also going to update the release process to make sure this doesn't happen again.

**Comment by breadbakerman** [comment]
No problem. All is well. 
Flatpack downgrade helped me out. I missed the --user flag and sudo'ed it though. This will need a manual cleanup.
I'll look out for 0.2.4

Awesome work.
